microprofile/messaging/src/test/java/io/helidon/microprofile/messaging/AssertableTestBean.java

```diff
@@ -20,10 +20,57 @@
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.hamcrest.Matcher;
 
 public interface AssertableTestBean {
 
     Set<String> TEST_DATA = new HashSet<>(Arrays.asList("teST1", "TEst2", "tESt3"));
 
     void assertValid();
+
+    default void await(String msg, CountDownLatch countDownLatch) {
+        try {
+            countDownLatch.await(500, TimeUnit.MILLISECONDS);
```

Comment:
`CountDownLatch.await` returns a boolean that tells you whether it succeeded, or timed out. That should be asserted to make sure the CDL completed.

Reply:
fixed